<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlay2Class</class>
 <widget class="QWidget" name="XPlay2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlay2</string>
  </property>
  <widget class="XVideoWidget" name="video">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openFile">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="XSlider" name="playPos">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>560</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="isPlay">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>510</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>xvideowidget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>XSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XPlay2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openFile</sender>
   <signal>clicked()</signal>
   <receiver>XPlay2Class</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlay</sender>
   <signal>clicked()</signal>
   <receiver>XPlay2Class</receiver>
   <slot>PlayOrPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderPressed()</signal>
   <receiver>XPlay2Class</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderReleased()</signal>
   <receiver>XPlay2Class</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>637</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>PlayOrPause()</slot>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
 </slots>
</ui>
